### DISCUSSION TITLE
Cannot access Web Panel @port: 64297

### DISCUSSION THREAD

**Discussion by tompaly** [telekom-security/tpotce#884] [Q&A]
I am able to reach the admin web panel via the 64294 port, however I've had no success reaching the regular web panel in multiple browsers.

**Comment by t3chn0m4g3** [comment]
Check with fail2ban if your IP was blocked.

**Reply by tompaly** [reply]
I am unable to manipulate fail2ban via the tsec user, is it possible to access as root?
![image](https://user-images.githubusercontent.com/31777461/131007622-a733d9d5-fb13-4c7b-8851-5f5156bfbc77.png)

**Reply by t3chn0m4g3** [reply]
Yes. `sudo su -`.
